tests/api/v1/subjects/put.js

```diff
@@ -231,78 +231,6 @@ describe(`api: PUT ${path}`, () => {
       done();
     });
   });
-
-  it('geolocation array must not contain null elements while patching',
-  (done) => {
-    const toPatch = {
-      isPublished: p1.isPublished,
-      name: p1.name,
-      geolocation: [null, null],
-    };
-    api.put(`${path}/${i0}`)
-    .set('Authorization', token)
-    .send(toPatch)
-    .expect(constants.httpStatus.BAD_REQUEST)
-    .end((err, res) => {
-      if (err) {
-        return done(err);
-      }
-
-      expect(res.body.errors[0].message).to
-      .equal('Expected type number but found type null');
-      expect(res.body.errors[0].source).to
-      .equal('geolocation');
-      done();
-    });
-  });
-
-  it('geolocation array cant contain less than two elements in patching',
-  (done) => {
-    const toPatch = {
-      isPublished: p1.isPublished,
-      name: p1.name,
-      geolocation: [1],
-    };
-    api.put(`${path}/${i0}`)
-    .set('Authorization', token)
-    .send(toPatch)
-    .expect(constants.httpStatus.BAD_REQUEST)
-    .end((err, res) => {
-      if (err) {
-        return done(err);
-      }
-
-      expect(res.body.errors[0].message).to
-      .equal('A non-null range must include two elements');
-      expect(res.body.errors[0].source).to
-      .equal('geolocation');
-      done();
-    });
-  });
-
-  it('geolocation array cant contain more than three elements in patching',
-  (done) => {
-    const toPatch = {
-      isPublished: p1.isPublished,
-      name: p1.name,
-      geolocation: [1,2,3],
-    };
-    api.put(`${path}/${i0}`)
-    .set('Authorization', token)
-    .send(toPatch)
-    .expect(constants.httpStatus.BAD_REQUEST)
-    .end((err, res) => {
-      if (err) {
-        return done(err);
-      }
-
-      expect(res.body.errors[0].message).to
-      .equal('A non-null range must include two elements');
-      expect(res.body.errors[0].source).to
-      .equal('geolocation');
-      done();
-    });
-  });
 });
 
```

Comment:
It was increased but due to some PR that is merged in master and then master merged it to this branch.